bundles/statistics/statsgrid2016/service/StateService.js

```diff
@@ -76,9 +76,9 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.StateService',
         selectRegion : function(region, componentId) {
```

Comment:
Should this be renamed toggleRegion or use with null region as parameter instead?